addons/website_project/controllers/main.py

```diff
@@ -1,38 +1,40 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo.http import request
-
-from odoo.addons.base.models.ir_qweb_fields import nl2br_enclose
+from odoo import _
+from odoo.addons.base.models.ir_qweb_fields import nl2br, nl2br_enclose
 from odoo.addons.website.controllers import form
+from odoo.tools import html2plaintext
 
 
 class WebsiteForm(form.WebsiteForm):
     def insert_record(self, request, model, values, custom, meta=None):
         if model.model == 'project.task':
+            visitor_sudo = request.env['website.visitor']._get_visitor_from_request()
+            visitor_partner = visitor_sudo.partner_id
+            if visitor_partner:
+                values['partner_id'] = visitor_partner.id
             # When a task is created from the web editor, if the key 'user_ids' is not present, the user_ids is filled with the odoo bot. We set it to False to ensure it is not.
             values.setdefault('user_ids', False)
-            if custom:
-                custom.replace('email_from', 'Email')
 
         res = super().insert_record(request, model, values, custom, meta=meta)
         if model.model != 'project.task':
             return res
         task = request.env['project.task'].sudo().browse(res)
+        custom = custom.replace('email_from', 'Email')
```

Comment:
Should be translated, I guess. 
```suggestion
        custom = custom.replace('email_from', _('Email'))
```

Reply:
🔄

Reply:
Let's do a fix in 17.2 and fix it here as well.